jenkins/Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 pipeline {
   agent {
         node {
-            label 'node:slave'
+            label 'project:any'
```

Comment:
probably should use the makerspace node, Ivan wants us to stop using the general slave and at some point will be getting rid of it. he hasn't notified us yet but this link shows which jobs use the general slave 
https://jenkins.wma.chs.usgs.gov/label/project:any/

Reply:
Ah ok.  I was just using the generic slave since I assume it's cheaper to run, since we requested extra resources for the Makerspace node.

Reply:
That's a good point, maybe we should request a smaller one for jobs that don't need the beefiness?

Reply:
Maybe there could be a makerspace_regular and makerspace_large or something, if they want to keep slaves tagged by team?  I assume they would want to minimize resource usage, but not sure how much of a priority it is